test/blackbox-tests/test-cases/doc-browser.t/run.t

```diff
@@ -0,0 +1,8 @@
+This tests shows how to use the `dune ocaml doc` command to open the
+documentation index to a browser.
+  $ export PATH=.:$PATH 
+  $ dune ocaml doc | sed -e 's/.*file:\/\/\([^ ]*\).*/\1/'
```

Comment:
You can use a different deliminator rather than `/` for sed so you don't have to escape so many characters. Something like # should work.